test/fixture/main.tf

```diff
@@ -40,7 +40,7 @@ module "aks" {
   vnet_subnet_id                  = azurerm_subnet.test.id
   os_disk_size_gb                 = 60
   enable_http_application_routing = true
```

Comment:
There are many "enable" variables that have the wrong naming convention. Does it makes sense to fix all the variables in a single PR ?

Reply:
Thanks for the tip, I think this is a special case since it's related to is single pr.
I agree with you, we need a single pr to fix all these `enable_xxx` variables.